lib/events/auditlog.go

```diff
@@ -789,17 +789,7 @@ func (l *AuditLog) GetSessionEvents(namespace string, sid session.ID, afterN int
 	if namespace == "" {
 		return nil, trace.BadParameter("missing parameter namespace")
 	}
-	// Print events are stored in the context of the downloaded session
-	// so pull them
-	if !includePrintEvents && l.ExternalLog != nil {
-		events, err := l.ExternalLog.GetSessionEvents(namespace, sid, afterN, includePrintEvents)
-		// some loggers (e.g. FileLog) do not support retrieving session only print events,
-		// in this case rely on local fallback to download the session,
-		// unpack it and use local search
-		if !trace.IsNotImplemented(err) {
-			return events, err
-		}
-	}
+
 	// If code has to fetch print events (for playback) it has to download
 	// the playback from external storage first
 	if err := l.downloadSession(namespace, sid); err != nil {
```

Comment:
While you're here, it would be great to rename `GetSessionEvents`. It doesn't work for  desktop sessions, as it attempts to convert into the legacy "chunks" format. I'm not sure if it works for kube/app/db sessions.

Reply:
@zmb3 I have reworked it, PTAL

Reply:
What's the code path taken by `tsh play --format=json <app session chunk ID>`? We store individual app access audit events in recordings, it would be even more confusing to read those through a function called `GetSSHSessionEvents` 🤔

Reply:
@espadolini you are right, it turns out that app access is using the same `GetSessionEvents`. So I think we should leave that name as it is, unless you have something better on mind to name it.